tests/e2e/convolution/BUILD.bazel

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The IREE Authors
```

Comment:
New files should set the copyright to the current year. Forked/moved files can keep the original year, but I think this should use the current year.

Reply:
Got it! Thanks.